src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -433,6 +433,61 @@ const char *smbios_mainboard_sku(void)
 	return sku;
 }
 
+static int fill_mainboard_smbios_type16(unsigned long *current, int *handle)
```

Comment:
@finus16 it can't be static, because ti will not be seen outside mainboard.c. You have to add __weak function `fill_mainboard_smbios_type16` in a header file, i.e. `smbios.h` and paste empty implementation in smbios.c like:

```
int __weak fill_mainboard_smbios_type16(unsigned long *current, int *handle) 
{
        return 0;
}
```

Reply:
I think it should be okay now.